api/swagger.yaml

```diff
@@ -350,6 +350,25 @@ paths:
         "200":
           description: "ok"
       x-swagger-router-controller: "Play"
+  /brick/setCoreLibrary/{brickId}/{isCore}:
+    put:
+      tags:
+      - "brick"
+      summary: "Marks a single notification as read."
```

Comment:
Should be something like "Sets whether a brick is in the core library or not."

Reply:
@william-gooch  done